auth_mellon_cache.c

```diff
@@ -601,7 +601,7 @@ void am_cache_env_populate(request_rec *r, am_cache_entry_t *t)
             apr_table_set(r->subprocess_env,prefixed_varname,value);
         }
 
-        if (d->merge_env_vars != 1) {
+        if (d->merge_env_vars == NULL || *d->merge_env_vars == '\0') {
```

Comment:
Under what circumstances will it be the empty string, and trigger the `[...] == '\0'`-condition?